packages/services/f-http/README.md

```diff
@@ -203,4 +163,4 @@ put | PUT a resource | resource URL _[string]_, body _[object]_, headers _[array
 patch | PATCH a resource | resource URL _[string]_, body _[object]_, headers _[array]_
 delete | DELETE a resource | resource URL _[string]_, headers _[array]_
 setAuthorisationToken | Set the authorisation token for all requests | authorisationToken _[string]_
-readConfiguration | Returns the configuration which has been applied | None
+readConfiguration | Returns the configuration which has been provided | None
```

Comment:
Would _Returns the current configuration._ or _Returns the active configuration._ be better?